app/models/busstop_node.rb

```diff
@@ -231,7 +237,7 @@ def self.findRoute(startID,endID)
 					"\n\n\n"
 				else currentNode == nil
 					puts "end of algo. Reached  Back to sourceNode."
-					return false		#end of algo. Reached  Back to sourceNode. ###Should Return pathRoute.###
+					return allRoutesFound		#end of algo. Reached  Back to sourceNode. ###Should Return pathRoute.###
```

Comment:
Line is too long. [101/80]<br>Unnecessary spacing detected.<br>Inconsistent indentation detected.<br>Missing space after #.<br>Tab detected.